app/services/resource_recommendation.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+class ResourceRecommendation
```

Comment:
`resources` and `initialize` (used by .new) are the only methods exposed to other classes. All other methods must be private